framework/src/main/java/org/tron/core/zen/ZksnarkInitService.java

```diff
@@ -57,6 +57,9 @@ private static String getParamsFile(String fileName) {
     } catch (IOException e) {
       logger.error(e.getMessage(), e);
     }
+    if (fileOut.exists()) {
+      fileOut.deleteOnExit();
+    }
```

Comment:
Will it delete twice ? It is possbile to delete these two files manually when close ?

Reply:
> Will it delete twice ?

No, Even that's okay.

> It is possbile to delete these two files manually when close ?

jni file, it's safer to delete  it at the end by `DeleteOnExitHook`.